components/application-authenticators/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/internal/SAMLSSOAuthenticatorServiceComponent.java

```diff
@@ -65,7 +65,7 @@ protected void activate(ComponentContext ctxt) {
             SAMLSSOAuthenticator samlSSOAuthenticator = new SAMLSSOAuthenticator();
             ctxt.getBundleContext().registerService(ApplicationAuthenticator.class.getName(), samlSSOAuthenticator, null);
             String postPagePath = CarbonUtils.getCarbonHome() + File.separator + "repository"
-                    + File.separator + "resources" + File.separator + "security" + File.separator
+                    + File.separator + "identity" + File.separator + "pages" + File.separator + "sso_responses" + File.separator
```

Comment:
"sso_responses" folder name is not appropriate. Because, samlsso_federate.html does not send a response - it sends a SAMLRequest to the federated IdP